bin/validate_ssa.py

```diff
@@ -6,17 +6,32 @@
 import tempfile
 import argparse
 import sys
+import coloredlogs
+import logging
 
 SSML_CWD = ".humvee"
 HUMVEE_URL = "https://repo.splunk.com/artifactory/maven-splunk-local/com/splunk/humvee-scala_2.11/1.2.1-SNAPSHOT/humvee-scala_2.11-1.2.1-20201022.220521-1.jar"
 
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+handler = logging.StreamHandler(sys.stdout)
+handler.setFormatter(coloredlogs.ColoredFormatter("%(asctime)s - %(levelname)s - %(message)s%(detail)s"))
```

Comment:
IMO the `detail` part is an unnecessary complication. If you want to log a bunch of events you can just
`logger.info('example logs:\n%s', '\n'.join(['line 1', 'line 2', 'line 3']))`